internal/datastore/postgres/migrations/zz_migration.0020_add_watch_api_index.go

```diff
@@ -1,20 +1,11 @@
 package migrations
 
-import (
-	"context"
-
-	"github.com/jackc/pgx/v5"
-)
-
-const watchAPIIndexToRelationTupleTable = `
-	relation_tuple (created_xid);`
+import "github.com/authzed/spicedb/internal/datastore/postgres/schema"
 
 func init() {
```

Comment:
It's not obvious why these migrations were refactored, but not _all migrations_ or _migrations in the other datastores_. I assume that you wanted to exercise the new `common.IndexDefinition` with a few migrations just to exercise the API. Please make this clear in the PR body.

Reply:
Yep

Reply:
I added the note to the PR body